tools/bin/hawqregister

```diff
@@ -432,7 +436,8 @@ class HawqRegister(object):
         if len(params[Format_FileLocations]['Files']):
             files, sizes = [params['DFS_URL'] + d['path'] for d in params[Format_FileLocations]['Files']], [d['size'] for d in params[Format_FileLocations]['Files']]
         encoding = params['Encoding']
-        self._set_yml_dataa(Format, files, sizes, params['TableName'], params['%s_Schema' % Format], params['Distribution_Policy'], params[Format_FileLocations], params['Bucketnum'], partitionby,\
+        bucketNum = params['Bucketnum'] if params['Distribution_Policy'].startswith('DISTRIBUTED BY') else 6
```

Comment:
I don't think the else should be "6", we need get the default bucket number from HAWQ (GUC: default_hash_table_bucket_number), or we don't need assign bucketnum here if we want to create table, i.e, don't specify in creating table DDL.